site/en/guides/thread-primer/node-roles-and-types.md

```diff
@@ -151,9 +151,200 @@ What you learned:
+      <div>When a new End Device is seeking to join the
+        Thread network.</div>
+      <div>Wrong. A Router cannot revert to an End Device in this scenario.
+      </div>
+    </div>
+    <div>
+      <div>When another device on the network elects to become a Router.
+      </div>
+      <div>Wrong.
+      </div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>Consider a scenario where a Thread network contains two 
+    groups of nodes that have radio connectivity with one another, but not
+    with members of the other group.
+    What conclusion can one draw from this?</div>
+    <div correct>
+        <div>There is more than one partition in that network.</div>
+        <div>Correct. A partition is formed around each group
+        of nodes that is able to communicate with one another. If there are
+        multiple groups of nodes that can communicate amongst themselves, but
+        not with members of other groups, then you know that these groups
+        each constitute a distinct partition.
+        </div>
+    </div>
+    <div>
+      <div>The network lost its Leader.</div>
+      <div>Incorrect.</div>
+    </div>
+    <div>
+      <div>All the Routers in the network have gone offline.</div>
+      <div>Incorrect. In that case, none of the nodes would be able to
+      communicate with one another.</div>
+    </div>    
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>What device is used to connect Thread and non-Thread networks?</div>
+    <div>
+      <div>a Gateway</div>
+      <div>Although the term 'Gateway', in traditional networking, does refer
+      to a device that connects two networks, there is a more accurate term.
+      </div>
+    </div>
+    <div correct>
+      <div>a Border Router</div>
+      <div>Correct. A Border Router is used to connect Thread and non-Thread
+        networks.</div>
+    </div>
+    <div>
+      <div>a Firewall</div>
+      <div>That is not right.</div>
+    </div>
+    <div>
+      <div>a Bridge</div>
+      <div>Incorrect. This term refers to a similar concept in traditional
+        networking referring to a device that connects two LANs that use the
+        same network protocol.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
+<div>
+  <devsite-multiple-choice>
+    <div>How many Leaders may a Thread network have?</div>
+    <div>
+      <div>Either none or exactly one.</div>
+      <div>This is wrong. A Thread network cannot have fewer than one Leader.
+      </div>
+    </div>
+    <div correct>
+      <div>One and only one.</div>
+      <div>Correct. A Thread network may have one and only one Leader.</div>
+    </div>
+    <div>
+      <div>More than one.</div>
+      <div>Wrong; a Thread network cannot have multiple Leaders.</div>
+    </div>
+  </devsite-multiple-choice>
+</div>
+
 ## License
 
-Copyright (c) 2021, The OpenThread Authors.
+Copyright (c) 2022, The OpenThread Authors.
```

Comment:
I think the copyright year for all the pages you're updating should be 2021-22, @jwhui can you confirm?